modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/DiscountedCumulativeGainAtTests.java

```diff
@@ -26,15 +26,14 @@
 import org.elasticsearch.index.shard.ShardId;
 import org.elasticsearch.search.SearchShardTarget;
 import org.elasticsearch.search.internal.InternalSearchHit;
-import org.elasticsearch.test.ESTestCase;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.concurrent.ExecutionException;
 
-public class DiscountedCumulativeGainAtTests extends ESTestCase {
+public class DiscountedCumulativeGainAtTests extends XContentRoundtripTests<DiscountedCumulativeGainAt> {
```

Comment:
Can XContentRoundtripTests be a small utility class that doesn't sit between ESTestCase and this test? In that case `roundtrip()` could be a static helper method. Just thinking about adding other shared test methods that some subtests need but others don't, this might get tricky with adding more levels of inheritance. Not a big deal though.